src/packages/client/src/__tests__/runtime-tests/blog-env-postgresql/index.js

```diff
@@ -219,17 +219,19 @@ module.exports = async () => {
         {
           array1key: 'array1value',
         },
-      ],
+      ]
     },
   })
 
   const result = await prisma.post.findMany({
     where: {
-      jsonData: [
-        {
-          array1key: 'array1value',
-        },
-      ],
+      jsonData: {
+        equals: [
+          {
+            array1key: 'array1value',
+          },
+        ]
+      },
     },
   })
```

Comment:
So only one query is a valid query?

I thought it was possible to do both and both should work?

Because we should test for both and probably add that to the MySQL / PostgeSQL integration tests I think

Reply:
No, we decided back when we fixed it, that only the `equals` version is valid, as you otherwise get into hells kitchen.
If you e.g. have a json with the content `equals: {}` you can't query it.

Reply:
Yes I remember now 😅 
So without the equals is producing an error now? I'm curious off its result.

Reply:
Yes without equals it's producing an error. I'll add a test case for that.